docs/contributing.md

```diff
@@ -16,91 +15,80 @@
 go run .
 ```
 
-## Run a full local setup
-
-### Setup
-
-* Install [Docker](https://docker.com/)
-  * (macOS, Windows) [Docker Desktop](https://www.docker.com/products/docker-desktop)
-  * (Linux) [Docker Engine](https://docs.docker.com/engine/install)
-* `envsubst`
-  * (macOS, Linux) `brew install gettext`
+## Run tests
 
----
+```bash
+go test ./...
 
-The local setup can be customized with environment varibles. Some are required for a functional deployment.
+# for shorter tests
+go test -short ./...
+```
 
-| Name             | Description                                                   | Default               |
-|------------------|---------------------------------------------------------------|-----------------------|
-| `NAMESPACE`      | Kubernetes namespace to install `infra`                       | `""`                  |
-| `VALUES`         | Values file to pass to Helm                                   | `docker-desktop.yaml` |
+## Linting
 
 ```bash
-make dev
+make tools
+make lint
 ```
 
-### Customizing local setup beyond the basics
+## Run in Docker Desktop
 
-If further customization is required, additional values files can be supplied to the installation by modifying the `VALUES` environment variable. It is recommended to append to `VALUES` rather than fully overriding it.
+### Prerequisites
 
-Some common configurations for local development include:
+Install [Docker Desktop](https://www.docker.com/products/docker-desktop/) and enable Kubernetes.
 
-* Disable persistence
+### Build and run
 
-```yaml
-# example infra.yaml
----
-server:
-  persistence:
-    enabled: false
+Run `make dev`:
+
+```bash
+make dev
 ```
 
-* Disable telemetry
+### Customize
 
-```yaml
-# example infra.yaml
----
-server:
-  config:
-    enable-telemetry: false
+The local Kubernetes setup uses [`helm`](https://helm.sh/) can be modified via a `values.yaml` file:
+
+```bash
+make dev flags="-f values.yaml"
 ```
 
-* Enable in-cluster connector
+Example `values.yaml` files:
 
-> Note: enabling in-cluster connector disables first-time user signup and requires an admin user to be created by Helm
+* Enable the in-cluster connector
 
 ```yaml
 ---
-# example infra.yaml
+# example values.yaml
 server:
-  additionalIdentities:
-    - name: admin
-      password: PASSWORD
+  config:
+    users:
+      - name: admin@local.dev
+        password: password
 
-  additionalGrants:
-    - user: admin
-      role: admin
-      resource: infra
+    grants:
+      - user: admin@local.dev
+        role: admin
+        resource: infra
 
 connector:
   enabled: true
+  config:
```

Comment:
Will this work without an access key?

Reply:
It should. If the connector is enabled in the same helm deployment as the server, and no access key is provided, one will be generated automatically